.github/workflows/ci.yml

```diff
@@ -146,11 +146,20 @@ jobs:
     - name: Show wgpu backend
       run: |
         python -c "from examples.tests.testutils import wgpu_backend; print(wgpu_backend)"
+    # We make sure to run the compare validation examples section first
+    # Since it seems that there is some dependency on the order in which
+    # the tests are run. It is unclear what is causing it but this order now
+    # matches the generation of the screenshots which seems to help stabilize
+    # the tests.
+    # https://github.com/pygfx/pygfx/pull/824
+    - name: Compare validation examples
+      run: |
+        pytest -s -v examples -k test_examples_compare
```

Comment:
I feel like you are so thorough @Korijn that I don't need to point you to this change but this is what resolved the issue for me......

I'm not sure if you have any ideas...

Reply:
Well, firstly you overlooked duplicating this part of the pipeline step:

```
      env:
        PYGFX_EXPECT_LAVAPIPE: true
```

Secondly, if the order matters, it's usually related to caching. Maybe the font manager evicts the font and reloads it, and then the SDFs generated ny freetype-py are different the second time around? Could be that there is some randomness in the SDF generation algorithm as a performance optimization.

Reply:
This is only used in one actual test that checks that asserts that the lava pipe pipeline is used. 

I'm happy to add it here and the regeneration worlflow